### PR TITLE
docs: add bridge-history deployment in readme

### PR DESCRIPTION
**Pull request by colinlyguo**
### PR title

Your PR title must follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) (as we are doing squash merge for each PR), so it must start with one of the following [types](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type):

- [x] docs: Documentation-only changes

### Deployment tag versioning

Has `tag` in `common/version.go` been updated or have you added `bump-version` label to this PR?

- [x] Yes

### Breaking change label

Does this PR have the `breaking-change` label?

- [x] No, this PR is not a breaking change

### CODE REVIEW

File: bridge-history-api/README.md [hunk 1]
```diff
@@ -79,3 +79,50 @@ provides REST APIs. Please refer to the API details below.
...
+   make bridgehistoryapi-api
+   ./build/bin/bridgehistoryapi-api
+   ```
+
+The endpoints provided in [./conf/config.json](./conf/config.json) are all public endpoints and have rate limits.
+
+For production usage:
+
+- For L1 endpoints, utilizing a service provider's free tier should suffice.
+- For L2 endpoints, consider [running a Scroll L2geth node](https://www.notion.so/scrollzkp/Mainnet-Deployment-Overview-c193b12cd8894e30a4ba77f6e97109a0) and using the exposed HTTP port.
```
Comment (0xmountaintop) [inline bridge-history-api/README.md]: use this https://scrollzkp.notion.site/Running-a-Scroll-L2geth-Node-Scroll-Mainnet-9d7b8aa810fc4cc4ae4add8b707a392d

File: bridge-history-api/README.md [hunk 2]
```diff
@@ -79,3 +79,50 @@ provides REST APIs. Please refer to the API details below.
...
+   ```
+   make bridgehistoryapi-api
+   ./build/bin/bridgehistoryapi-api
+   ```
+
+The endpoints provided in [./conf/config.json](./conf/config.json) are all public endpoints and have rate limits.
+
+For production usage:
+
+- For L1 endpoints, utilizing a service provider's free tier should suffice.
```
Comment (georgehao) [inline bridge-history-api/README.md:127]: add a link: how to run l1

File: bridge-history-api/README.md (hunk 1 above)
Comment (isabellewei) [reply bridge-history-api/README.md]: we also have public docs for this now here: https://docs.scroll.xyz/en/developers/guides/running-a-scroll-node/

File: bridge-history-api/conf/config.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
 		"USDCGatewayAddr": "0xf1AF3b23DE0A5Ca3CAb7261cb0061C0D779A5c7B",
 		"LIDOGatewayAddr": "0x6625C6332c9F91F2D27c304E729B86db87A3f504",
 		"DAIGatewayAddr": "0x67260A8B73C5B77B55c1805218A42A7A6F98F515",
+		"PufferGatewayAddr": "0xA033Ff09f2da45f0e9ae495f525363722Df42b2a",
```
Comment (isabellewei) [inline bridge-history-api/conf/config.json:18]: Is this on purpose, to have two different addresses for `PufferGatewayAddr`??

File: bridge-history-api/README.md (hunk 2 above)
Comment (isabellewei) [reply bridge-history-api/README.md:127]: I think this is ok, no need to explain how to get Ethereum L1 endpoint

File: bridge-history-api/README.md (hunk 1 above)
Comment (colinlyguo) [reply bridge-history-api/README.md]: fixed in [3a9aaa2](https://github.com/scroll-tech/scroll/pull/1281/commits/3a9aaa2be943f64f0d93f4463a8c174e69b9b536), used the [docs.scroll.xyz](https://docs.scroll.xyz/en/developers/guides/running-a-scroll-node) one.

File: bridge-history-api/README.md (hunk 2 above)
Comment (colinlyguo) [reply bridge-history-api/README.md:127]: free tiers of infura, alchemy, or other service providers are OK.

**Review by 0xmountaintop — APPROVED** [review]

**Review by georgehao — APPROVED** [review]

**Pull request merged**